modules/installation-about-disconnected.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_disconnected/installing-disconnected-preparations.adoc
+
+[id="installation-about-disconnected_{context}"]
+= About disconnected installations
+
+In {product-title} {version}, you can perform an installation that does not
+require an active connection to the internet to obtain software components. You
+complete a disconnected installation on only infrastructure that you provision,
+not infrastructure that the installer provisions, so your platform selection is
+limited.
+// maybe point out that you can follow the bare metal installation rules on supported hardware and link to the matrix
+
+If you choose to perform a disconnected installation on a cloud platform, you
+still require access to its cloud APIs and will still require internet access
+to use your cluster. Depending on your network, you might require less internet
+access for an installation on bare metal hardware or on VMware vSphere.
+
+To complete a disconnected installation, you must create a repository that
+mirrors the contents of the {product-title} registry and contains the
+installation media. You can create this mirror on a bastion host, which can
+access both the internet and your closed network, or by using other methods
+that meet your restrictions.
+
+[IMPORTANT]
+====
+Because disconnected installations are always on user-provisioned infrastructure,
+always complete a standard user-provisioned infrastructure installation before
+you attempt a disconnected installation.
+====
+
+[id="installation-disconnected-limits{context}"]
+== Additional limits
+
+Disconnected clusters have the following additional limitations and restrictions:
+
+* Cluster settings always displays errors because the cluster cannot determine
+if updates are available.
+* The authentication Operator might randomly fail.
+* Because you cannot access the required ImageStreamTags, you cannot use the
+contents of the Developer Catalog even if it contains items.
+* The `TelemeterClientDown` and `Watchdog` alerts from the monitoring Operator
+always display.
+
+[id="installation-disconnected-mirror_{context}"]
+== About the mirror repository
+
+The key component that is require to complete a disconnected installation is
+the mirror repository. This repository contains both the installation media
+and the images that the cluster requires.
+
+Because of the way that {product-title} verifies integrity for the release
+payload, the image references in your local registry are identical to the ones
+that are hosted on
+link:
+During the bootstrapping process of installation, the images must have the same
+names no matter which repository they are pulled from.
+To ensure that the release payload is identical, you mirror the repository.
+
+You must specify separate mirrors for both the images and the release package itself.
```

Comment:
You can use the same mirror for both the release image and the secondard images referenced from the release image (which is what [the installer docs][1] do).

[1]: https://github.com/openshift/installer/blob/master/docs/user/customization.md#image-content-sources